### PR TITLE
Add more clarification around data tiering.

### PR DESCRIPTION
**Pull request by gayyappan**
# Description

[Short summary of why you created this PR]

# Links

Fixes #[insert issue link, if any]

# Writing help

For information about style and word usage, see the [style guide](https://docs.timescale.com/about/latest/contribute-to-docs/)

# Review checklists

Reviewers: use this section to ensure you have checked everything before approving this PR:

## Subject matter expert (SME) review checklist

*   [ ] Is the content technically accurate?
*   [ ] Is the content complete?
*   [ ] Is the content presented in a logical order?
*   [ ] Does the content use appropriate names for features and products?
*   [ ] Does the content provide relevant links to further information?

## Documentation team review checklist

*   [ ] Is the content free from typos?
*   [ ] Does the content use plain English?
*   [ ] Does the content contain clear sections for concepts, tasks, and references?
*   [ ] Have any images been uploaded to the correct location, and are resolvable?
*   [ ] If the page index was updated, are redirects required
      and have they been implemented?
*   [ ] Have you checked the built version of this content?

### CODE REVIEW

File: use-timescale/data-tiering/about-data-tiering.md [hunk 1]
```diff
@@ -11,9 +11,10 @@ cloud_ui:
 
 # About data tiering
 
-Save on storage costs by tiering rarely used data to a low-cost object storage layer.
+Save on storage costs by tiering rarely used data to a low-cost object storage layer
+on Timescale Cloud.
```
Comment (iroussos) [inline use-timescale/data-tiering/about-data-tiering.md]: I propose to skip this change and add an info box similar to what we have on `/use-timescale/data-tiering/index.md`

File: use-timescale/data-tiering/about-data-tiering.md [hunk 2]
```diff
@@ -11,9 +11,10 @@ cloud_ui:
...
 
-Timescale includes traditional disk storage, and a low-cost object storage
+Timescale Cloud service includes traditional disk storage, and a low-cost object storage
```
Comment (iroussos) [inline use-timescale/data-tiering/about-data-tiering.md]: Let's go with he following and I'll fix this on the follow-up with the name changes and the positioning

```suggestion
Timescale services include traditional disk storage, and a low-cost object storage
```

**Review by iroussos — APPROVED** [review]
Thank you @gayyappan, looks good to me, I added 2 minor requests only.

File: use-timescale/data-tiering/about-data-tiering.md (hunk 2 above)
Comment (gayyappan) [reply use-timescale/data-tiering/about-data-tiering.md]: @iroussos I highlighted Cloud to differentiate between MST and Timescale Cloud. Are we using soem other nomenclature now?

Comment (iroussos) [reply use-timescale/data-tiering/about-data-tiering.md]: Ok, let's leave this as is to differentiate from MST.
In general we do not use Timescale Cloud any more, just Timescale, but it makes sense in this case

**Pull request merged**